<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>545</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="but1">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>470</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but2">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>470</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but6">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>470</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but3">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>470</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but4">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>470</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but5">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>470</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but8">
    <property name="geometry">
     <rect>
      <x>725</x>
      <y>470</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but7">
    <property name="geometry">
     <rect>
      <x>625</x>
      <y>470</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card1">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>80</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>card1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card4">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>350</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>card4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>260</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>card3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>170</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>card2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card5">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>440</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>card5</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
